City-of-Helsinki/helsinki-paatokset
# Pp 313 misc fixes

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-313

**How to test**

Checkout branch and run `drush cim -y; drush cr`

Create two pages (or if you have existing ones) set one as a parent page and the other as a child. Test that the sidebar navigations work and also test what happens in some new sidebar content is added. The content should appear after the sidebar navigation.

Test with a page that has a custom block in the content area i.ex. with https://helsinki-paatokset.docker.so/fi/tietoa-paatoksenteosta

- Add some content in the upper content area, this should appear before the decision tree and inside the main content.
- Make the sidebar visible in the page, the content in the main content area should be accordingly with the sidebar
- Add some content to the lower content area, this should appear after the main content as full width

Go to some policymaker page that doesn't have any content i.ex https://helsinki-paatokset.docker.so/fi/paattajat/psykososiaalisen-tyon-paallikko-paihdeavokuntoutus

There should be a text "Tällä päättäjällä ei ole päätöksiä" in the policymaker landinpage as well as in the documents if there aren't any document to show.

(Also fixed some error with divs in the agendas submenu block)
